src/main/resources/assets/chisel/lang/es_es.lang

```diff
@@ -0,0 +1,2871 @@
+item.chisel.chisel_iron.name=Cincel
+item.chisel.chisel_diamond.name=Cincel de Diamante
+item.chisel.chisel_hitech.name=iChisel
+item.chisel.offsettool.name=Varita de Diseños Ender
+
+chisel.spawnwither=Un Cincel puede ser algo peligroso...
+
+item.chisel.chisel.desc.gui=%sClic derecho%s para abrir la Interfaz
+item.chisel.chisel.desc.lc1=%sClic izquierdo%s para cincelar un bloque del mundo
+item.chisel.chisel.desc.lc2=%sSelecciona el objeto%s al tenerlo en el inventario
```

Comment:
I'd replace "objeto" with "bloque" (originally "block") and "al tenerlo" with "colocándolo" (originally "by leaving it")
(Also propagate this change to L51, L54 & L60)

Reply:
Ohhh, thanks for those detail, i´ll change the Ender Offset Wand by "Varita de Desplazamiento de Ender",

and the second detail i´ll leave it as you modified.

Thanks a lot for inspecting!

Reply:
Also, i make the changes that you mentioned it, so now the file has all the changes